savant/client/frame_source/image_source.py

```diff
@@ -148,8 +152,30 @@ def _update_param(self, name, value) -> 'JpegSource':
 
     def __repr__(self):
         return (
-            f'JpegSource('
+            f'{self.__class__.__name__}('
             f'source_id={self._source_id}, '
             f'file={self._file}, '
             f'pts={self._pts})'
         )
+
+
+class JpegSource(ImageSource):
```

Comment:
Do we need to have different classes for JPEG and PNG sources? They are the same anyway and there's no validation the file type matches the class.

Reply:
@tomskikh the idea is to make the use self-explanatory. Please keep as is.